docs/DEPLOY_DEMO.md

```diff
@@ -2,137 +2,200 @@
 ```
-$ ls
-README.md  cdk.json ...
+
+5. Then make sure to remove the instance profile, role, and policy that are no longer needed.
+
+```console
+aws iam detach-role-policy --role-name packer-instance-role --policy-arn "arn:aws:iam::$PACKER_ACCOUNT_ID:policy/packer-instance-profile-policy"
+aws iam remove-role-from-instance-profile --instance-profile-name packer-instance-profile --role-name packer-instance-role
+aws iam delete-role --role-name packer-instance-role
+aws iam delete-policy --policy-arn "arn:aws:iam::$PACKER_ACCOUNT_ID:policy/packer-instance-profile-policy"
+aws iam delete-instance-profile --instance-profile-name packer-instance-profile
 ```
 
-2. Edit around line 22 of lib/eks-cluster-stack.ts to set access rights for an existing role or user to the EKS cluster. By default, a role named Admin is the target.
+## Deploying the CDK App
+1. We will no deploy the CDK App, which includes the EKS Cluster and Docker container images.
 
-If you prefer to set this without using CDK, refer to [Enabling IAM principal access to your cluster](https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html)  for guidance.
+    If you are currently in the `packer` directory, return to the directory above. Ensure that there is a `cdk.json` file in the same directory.
 
+```console
+ls
+ README.md  cdk.json ...
 ```
-// Configure aws-auth to allow IAM Role for Admin to access EKS Cluster
-// https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_eks-readme.html#permissions-and-security
-const adminRole = iam.Role.fromRoleName(this, 'AdminRole', 'Admin') <= HERE
-cluster.awsAuth.addMastersRole(adminRole)
 
-// For user
-// const user = iam.User.fromUserName(this, 'User', 'username')
-// cluster.awsAuth.addUserMapping(user, { groups: ['system:masters'] })
-```
+2. Open the file [containers/pixel-streaming/Dockerfile](../containers/pixel-streaming/Dockerfile) and replace the `UNREAL_ENGINE_APP` environment variable with the name of the `.sh` file used to launch your Unreal Engine application, for example `ThirdPersonShooter.sh`.
 
-3. Modify the allowedIpRange around line 39 of `lib/eks-cluster-stack.ts` to the IP range that you want to allow access to the demo.
-   To allow access from all IP addresses, specify `0.0.0.0/0`.
-```
-const allowedIpRange = ec2.Peer.ipv4('59.138.151.0/24') <= HERE
-```
+3. Open the [lib/eks-cluster-stack.ts](../lib/eks-cluster-stack.ts) TypeScripts CDK file and update the deployment specific variables.
 
-4. Change the imageId around line 67 of `lib/eks-cluster-stack.ts` to the ID of the AMI you created in the previous step.
-```
-const launchTemplate = new ec2.CfnLaunchTemplate(
-      this,
-      'NodeLaunchTemplate',
-      {
-        launchTemplateData: {
-          imageId: 'ami-038246843a7aabe29',  <= HERE
-          instanceType: 'g4dn.xlarge',
-          ...
+    a) `EKS_ACCESS_ROLE` 
```

Comment:
It would be good to add a note about the modifications when using IAMUser and when using IAMRole.

Reply:
Not sure I understand what you would like to be included in the note. There is a description on row 90 with a link to the docs.